docs/modules/mssql.md

```diff
@@ -0,0 +1,65 @@
+# MSSQLServer
+
+Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
+
+## Introduction
+
+The Testcontainers module for MSSQLServer.
+
+## Adding this module to your project dependencies
+
+Please run the following command to add the MSSQLServer module to your Go dependencies:
+
+```
+go get github.com/testcontainers/testcontainers-go/modules/mssql
+```
+
+## Usage example
+
+<!--codeinclude-->
+
+[Creating a MSSQLServer container](../../modules/mssql/examples_test.go) inside_block:runMSSQLServerContainer
+
+<!--/codeinclude-->
+
+!!! warning "EULA Acceptance"
```

Comment:
Microsoft requires users to accept the EULA before using the MS SQL Server, so the module implementation will require users to manually accept, similar fashion to the [java](https://github.com/testcontainers/testcontainers-java/blob/main/modules/mssqlserver/src/main/java/org/testcontainers/containers/MSSQLServerContainer.java#L88) implementation